charts/tidb-cluster/templates/_helpers.tpl

```diff
@@ -118,10 +118,20 @@ config-file: |-
 {{/*
 Encapsulate pump configmap data for consistent digest calculation
 */}}
+{{- define "pump.tlsSecretName" -}}
+{{ .Values.clusterName }}-{{ .Release.Namespace }}-pump
+{{- end -}}
+
 {{- define "pump-configmap.data" -}}
 pump-config: |-
     {{- if .Values.binlog.pump.config }}
 {{ .Values.binlog.pump.config | indent 2 }}
+    {{- if .Values.enableTLSCluster }}
+  [security]
+  ssl-ca = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+  ssl-cert = "/var/lib/pump-tls/cert"
+  ssl-key = "/var/lib/pump-tls/key"
+    {{- end -}}
```

Comment:
The configuration rendered by contoroller should also be updated.

Reply:
done